blocks/api/raw-handling/index.js

```diff
@@ -57,8 +57,14 @@ export default function rawHandler( { HTML, plainText = '', mode = 'AUTO' } ) {
 		HTML = converter.makeHtml( plainText );
 	}
 
+	const shortcodesConverted = shortcodeConverter( HTML );
+
+	//if the shortcodeConverter returns just one element it has no shortcodes
+	//even if we paste only one shortcode it returns more than one element because empty strings are added
```

Comment:
Comment format should be:

```
// If the …
// even if
```

i.e., capitalized sentences and a leading space for each line. Bonus if lines are limited in length (my preference is 78/80 chars, but commonly we see 100 as a hard limit).

Reply:
As for the comment itself, I'd make it terser, e.g. _shortcodeConverter will always return more than one element if shortcodes are matched, as empty HTML strings are included_.